components/eamxx/src/control/atmosphere_driver.cpp

```diff
@@ -414,6 +422,19 @@ void AtmosphereDriver::setup_column_conservation_checks ()
   m_atm_process_group->setup_column_conservation_checks(conservation_check, fail_handling_type);
 }
 
+void AtmosphereDriver::setup_shoc_tms_links ()
+{
+  EKAT_REQUIRE_MSG(m_atm_process_group->has_process("tms"),
+                   "Error! Attempting to setup link between "
+                   "SHOC and TMS, but TMS is not defined.\n");
+  EKAT_REQUIRE_MSG(m_atm_process_group->has_process("shoc"),
+                   "Error! Attempting to setup link between "
+                   "SHOC and TMS, but SHOC is not defined.\n");
+
+  auto shoc_process = m_atm_process_group->get_process_nonconst("shoc");
+  shoc_process->get_params().set<bool>("apply_tms", true);
```

Comment:
This is the only use I can find for `get_params()`, so I changed it to be nonconst.